src/spec-node/collectionCommonUtils/packageCommandImpl.ts

```diff
@@ -0,0 +1,140 @@
+import tar from 'tar';
+import { PackageCommandInput } from './package';
+import { isLocalFile, isLocalFolder, mkdirpLocal, readLocalDir, readLocalFile, rmLocal } from '../../spec-utils/pfs';
+import { LogLevel } from '../../spec-utils/log';
+import path from 'path';
+
+export interface SourceInformation {
+	source: string;
+	owner?: string;
+	repo?: string;
+	tag?: string;
+	ref?: string;
+	sha?: string;
+}
+
+export const OCICollectionFileName = 'devcontainer-collection.json';
```

Comment:
I think style-wise typescript constants should be BIG_LIKE_THIS

Reply:
I tried to match the casing with the already existing constants 😅 

![image](https://user-images.githubusercontent.com/24955913/193157246-d1e7c4a1-dc9a-4d6e-bb9c-9a4f375fba1d.png)

we do have some caps with _ variables but only for Features stuff 🤔 

![image](https://user-images.githubusercontent.com/24955913/193157324-5233ec5e-85af-4b2d-a4e7-960230bc5b78.png)

I don't have a preference though.

Reply:
Camel-case is fine. I often lower-case abbreviations when they are at the start (but upper-case in the middle). E.g., `ociCollectionFileName`, but I don't have a strong preference on that.